### DISCUSSION TITLE
Spotify songs

### DISCUSSION THREAD

**Discussion by ivory-uwu** [Eloston/ungoogled-chromium#2851] [Q&A]
I just installed the WindVine CDM extension, and when I play songs in spotify, it skips my entire playlist saying that "

Spotify can't play this right now. If you have the file on your computer you can import it."

**Comment by PF4Public** [comment]
Please [search existing and closed issues](https://github.com/ungoogled-software/ungoogled-chromium/issues?q=is%3Aissue+Spotify) for similar ones first
